### DISCUSSION TITLE
Is it possible to customize Window's title font and size?

### DISCUSSION THREAD

**Discussion by HumanMus1c** [ramensoftware/windhawk-mods#680] [Mod ideas]
Why Windows not allow any software to change the title font of windows?

**Comment by m417z** [comment]
There's no Windhawk mod for that currently

**Comment by Anixx** [comment]
Do you mean like this?
![изображение](https://github.com/ramensoftware/windhawk-mods/assets/1271736/0d75fb14-ce8a-45b6-b3a2-9750b8e4fed8)
Yes, you can.
